presto-main/src/main/java/com/facebook/presto/operator/repartition/OptimizedPartitionedOutputOperator.java

```diff
@@ -122,7 +122,7 @@ public OptimizedPartitionedOutputOperator(
                 operatorContext);
 
         operatorContext.setInfoSupplier(pagePartitioner.getPartitionedOutputInfoSupplier());
-        this.systemMemoryContext = operatorContext.newLocalSystemMemoryContext(PartitionedOutputOperator.class.getSimpleName());
```

Comment:
Having `OptimizedPartitionedOutputOperator` and `PartitionedOutputOperator` merged into the same allocation tag may have been intentional. Not sure who would mind these being broken out separately though.